src/EnergyPlus/UnitarySystem.cc

```diff
@@ -258,6 +283,33 @@ namespace UnitarySystems {
                               int const ZoneOAUnitNum,
                               Real64 const OAUCoilOutTemp,
                               bool const ZoneEquipment)
+    {
+        Real64 sysOutputProvided = 0.0;
+        Real64 latOutputProvided = 0.0;
+        simulateSys(Name,
+                    FirstHVACIteration,
+                    AirLoopNum,
+                    CompIndex,
+                    HeatActive,
+                    CoolActive,
+                    ZoneOAUnitNum,
+                    OAUCoilOutTemp,
+                    ZoneEquipment,
+                    sysOutputProvided,
+                    latOutputProvided);
+    }
+
+    void UnitarySys::simulateSys(std::string const &Name,
```

Comment:
and each of those functions above now call the original simulate routine that now passes sysOutputRequired and latOutputRequired

Reply:
Also, this was a clever approach.

Reply:
@mjwitte suggested a new function with a different signature. I thought about that and this is what I came up with. Clever maybe, but if there is a better way I am always willing to learn. Thanks for the review.